tests/test_data.py

```diff
@@ -50,27 +50,45 @@ def test_user_s3_bucket_no_id_provided(self):
         bucket = dallinger.data.user_s3_bucket()
         assert bucket
 
+    @pytest.mark.skip
     def test_data_loading(self):
         data = dallinger.data.load("3b9c2aeb-0eb7-4432-803e-bc437e17b3bb")
         assert data
         assert data.networks.csv
 
-    def test_register_id(self):
+    def test_register_id_disabled(self, active_config):
         new_uuid = "12345-12345-12345-12345"
         url = dallinger.data.register(new_uuid, "http://original-url.com/value")
+        assert url is None
 
-        # The registration creates a new file in the dallinger-registrations bucket
-        assert url.startswith("https://dallinger-registrations.")
-        assert new_uuid in url
-
-        # These files should be inaccessible to make it impossible to use the bucket
-        # as a file repository
-        res = requests.get(url)
-        assert res.status_code == 403
-
-        # We should be able to check that the UUID is registered
-        assert dallinger.data.is_registered(new_uuid) is True
-        assert dallinger.data.is_registered("bogus-uuid-value") is False
+    def test_register_id(self, active_config):
+        new_uuid = "12345-12345-12345-12345"
+        active_config.set("enable_global_experiment_registry", True)
+        with mock.patch("dallinger.data.boto3") as boto:
```

Comment:
Ugh. At some point (not now), we need to write a wrapper around bucket access to avoid having to mock such a complex interface.